source/geometry_model/ellipsoidal_chunk.cc

```diff
@@ -600,6 +600,17 @@ namespace aspect
 
     template <int dim>
     double
+    EllipsoidalChunk<dim>::height_above_reference_surface(const Point<dim> &position) const
+    {
+      AssertThrow(false, ExcMessage("Function height_above_reference_surface is not yet implemented"
+                                    "for the ellipsoidal chunk geometry model."
+                                    "Consider using a box, spherical shell, or chunk.") );
```

Comment:
C++ takes successive `"quote enclose" "d strings"` and just concatenates them. Here, this means that you'll get an error message that doesn't have a space after `implemented` and `for the ellipsoidal`, and the same between the second and third line. Can you just add a space at the end of the first and second lines?